File: src/main/java/com/fwdekker/randomness/decimal/DecimalSettingsAction.java
package com.fwdekker.randomness.decimal;

import com.fwdekker.randomness.SettingsAction;
import com.fwdekker.randomness.SettingsDialog;


/**
 * Controller for random decimal generation settings.
 */
public final class DecimalSettingsAction extends SettingsAction {
    @Override
    public SettingsDialog createDialog() {
        return new DecimalSettingsDialog();
    }

    @Override
    public String getTitle() {
        return "Random Decimal Settings";
    }
}

File: src/main/kotlin/com/fwdekker/randomness/decimal/DecimalSettingsAction.kt
package com.fwdekker.randomness.decimal

import com.fwdekker.randomness.SettingsAction


/**
 * Controller for random decimal generation settings.
 */
class DecimalSettingsAction : SettingsAction() {
    override val title = "Decimal Settings"


    public override fun createDialog() = DecimalSettingsDialog()
}
